probe/core/src/main/client/app.js

```diff
@@ -407,6 +412,52 @@ Probe.InvocationDetailRoute = Ember.Route.extend(Probe.ResetScroll, {
     }
 });
 
+Probe.EventsRoute = Ember.Route.extend(Probe.ResetScroll, {
+    queryParams : {
+        page : {
+            refreshModel : true
+        },
+        kind : {
+            refreshModel : true
+        }
+    },
+    setupController : function(controller, model) {
+        this._super(controller, model);
+        controller.set("pages", buildPages(model.page, model.lastPage));
+    },
+    model : function(params) {
+        var query = '', filters = '', page = '';
+        filters = appendToFilters(filters, 'eventInfo', params.eventInfo);
+        filters = appendToFilters(filters, 'type', params.type);
+        filters = appendToFilters(filters, 'qualifiers', params.qualifiers);
+        if (params.kind) {
+            filters = appendToFilters(filters, 'kind', params.kind);
+        }
+        query = appendToQuery(query, 'filters', filters);
+        if (params.page) {
+            query = appendToQuery(query, 'page', params.page);
+        }
+        return $.getJSON(restUrlBase + 'events' + query).done(function(data) {
+            data.transformed = getRootNode(data, null);
```

Comment:
The transformation is not needed here. I would guess that `getRootNode()` invocation fails anyway.